src/main/java/de/uhd/ifi/se/decision/management/jira/model/KnowledgeElement.java

```diff
@@ -108,7 +110,11 @@ public KnowledgeElement(Issue issue) {
 		this.key = issue.getKey();
 		this.documentationLocation = DocumentationLocation.JIRAISSUE;
 		this.creationDate = issue.getCreated();
```

Comment:
Can you add the creation date and author name as the first date/author in the treemap and get rid of these attributes? They seem to be not necessary anymore.